cmake/SSGCommon.cmake

```diff
@@ -129,11 +129,12 @@ macro(ssg_build_shorthand_xml PRODUCT)
     )
 endmacro()
 
-macro(ssg_build_xccdf_unlinked PRODUCT)
+macro(ssg_build_xccdf_unlinked PRODUCT DISA_STIG_VERSION)
     add_custom_command(
         OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/xccdf-unlinked-resolved.xml"
         COMMAND "${XSLTPROC_EXECUTABLE}" --stringparam ssg_version "${SSG_VERSION}" --output "${CMAKE_CURRENT_BINARY_DIR}/xccdf-unlinked-resolved.xml" "${CMAKE_CURRENT_SOURCE_DIR}/transforms/shorthand2xccdf.xslt" "${CMAKE_CURRENT_BINARY_DIR}/shorthand.xml"
         COMMAND "${OPENSCAP_OSCAP_EXECUTABLE}" xccdf resolve -o "${CMAKE_CURRENT_BINARY_DIR}/xccdf-unlinked-resolved.xml" "${CMAKE_CURRENT_BINARY_DIR}/xccdf-unlinked-resolved.xml"
+        COMMAND "${SSG_SHARED_UTILS}/add_stig_references.py" --disa-stig "${SSG_SHARED_REFS}/disa-stig-${PRODUCT}-${DISA_STIG_VERSION}-xccdf-manual.xml" --unlinked-xccdf "${CMAKE_CURRENT_BINARY_DIR}/xccdf-unlinked-resolved.xml"
```

Comment:
What if `add_stig_references.py` just received the `SSG_SHARED_REFS` and `PRODUCT` and it tried to figure out by itself the latest `DISA_STIG_VERSION`?

Reply:
@yuumasato output right now would only be XCCDF and DS files.

> What if `add_stig_references.py` just received the `SSG_SHARED_REFS` and `PRODUCT` and it tried to figure out by itself the latest `DISA_STIG_VERSION`?

Not a bad idea. If you could do something like `*` or regex in the file name. e.g.
`"${SSG_SHARED_REFS}/disa-stig-${PRODUCT}-*-xccdf-manual.xml"`

We don't use multiple versions for references files for the same product, and with using `${PRODUCT}` should get the right reference file for the product if `xccdf-manual.xml` is included as well. Plus, it does get annoying to have to change reference versions everywhere.